freqtrade/rpc/rest_client.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+"""
+Simple command line client into RPC commands
+Can be used as an alternate to Telegram
+"""
+
+from requests import get
+from sys import argv
+
+if len(argv) == 1:
```

Comment:
consider using `argparse` as we do in the rest of the bot

it makes handling of command line arguments much easier than `argv[2] == "whatever"` .